quad_pyblish_module/plugins/tvpaint/publish/extract_psd.py

```diff
@@ -64,7 +64,7 @@ def process(self, instance):
                 george_script_lines.append(
                     "tv_clipsavestructure \"{}\" \"PSD\" \"image\" {}".format(
                         dst_filepath,
-                        int(new_filename) - 1
+                        int(instance.context.data["sceneMarkIn"]) + offset
```

Comment:
IMO the `instance.context.data["sceneMarkIn"]` should be retrieved only once and stored in a variable before the loop, to avoid memory access / load in ram of the data for every iteration, yeah its Python and performance / memory usage isn't the key element but its good pratice.